qadence/backends/jax_utils.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import annotations
+
+from typing import Any
+
+import jax.numpy as jnp
+from jax import Array, device_get
+from sympy import Expr
+from sympy2jax import SymbolicModule as JaxSympyModule
+from torch import Tensor, cdouble, from_numpy
+
+from qadence.types import ParamDictType
+
+
+def jarr_to_tensor(arr: Array, dtype: Any = cdouble) -> Tensor:
+    return from_numpy(device_get(arr)).to(dtype=dtype)
```

Comment:
Are you sure you need a `device_get` here? Usually you only use that when you're explicitly doing things...

Reply:
i think i added it initially because stuff was failing without :D i can check it

Reply:
yes, im sure we need it: ` TypeError: expected np.ndarray (got ArrayImpl)`